plugins/obs-ffmpeg/obs-ffmpeg-mpegts.c

```diff
@@ -327,11 +327,50 @@ static inline int connect_mpegts_url(struct ffmpeg_output *stream, bool is_rist)
 					"Can not allocate memory.");
 		goto fail;
 	}
+	/* For SRT, pass streamid & passphrase; for RIST, pass passphrase, username
+	 * & password.
```

Comment:
You should use `*` if the `/*` is `/**`.

Reply:
Please read our contribution and code-style guidelines first: https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst